drivers/network/afd/afd/read.c

```diff
@@ -177,14 +177,16 @@ static NTSTATUS ReceiveActivity( PAFD_FCB FCB, PIRP Irp ) {
                            RecvReq->BufferCount, FALSE );
             if (FCB->Overread && FCB->LastReceiveStatus == STATUS_SUCCESS)
```

Comment:
The new test in `ws2_32:recv` confirms that this special behavior is never needed, and this is the only use of the `Overread` member. Please remove it entirely.